website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB data platform setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Crate.io, Inc.
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Not Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'Community Forum'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/no-configs'
+---
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta}/>
+
+
+[CrateDB] is compatible with PostgreSQL, so its dbt adapter strongly depends on
+dbt-postgres, documented at [PostgreSQL profile setup].
+
+CrateDB targets are configured exactly the same way, see also [PostgreSQL
+configuration], with just a few things to consider which are special to
+CrateDB. Relevant details are outlined at [using dbt with CrateDB],
+which also includes up-to-date information.
+
+
+## Profile configuration
+
+CrateDB targets should be set up using a configuration like this minimal sample
+of settings in your [`profiles.yml`] file.
```

Comment:
There is currently a hiccup.

![image](https://github.com/user-attachments/assets/59277975-0336-47d4-b1c7-75f3e778af39)

Reply:
Hm. I think all the links worked well before.

![image](https://github.com/user-attachments/assets/cb57b093-f531-41b3-8967-46b80b6e52c4)

Maybe I did a mistake, misunderstanding your advises.

Reply:
Let me revent the changes.

Reply:
I think it is rendering well now?

-- https://docs-getdbt-com-git-fork-crate-workbench-cratedb-dbt-labs.vercel.app/docs/core/connect-data-platform/cratedb-setup

Reply:
they do work! thanks @amotl !